src/python/WMCore/WMSpec/WMWorkload.py

```diff
@@ -1483,7 +1483,7 @@ def setSubscriptionInformation(self, initialTask=None, custodialSites=None,
                                    custodialGroup="DataOps", nonCustodialGroup="DataOps",
                                    priority="Low", primaryDataset=None,
                                    useSkim=False, isSkim=False,
-                                   dataTier=None, deleteFromSource=False):
+                                   dataTier=None, deleteFromSource=False, dataset_lifetime=None):
```

Comment:
Can you please keep the same variable name convention defined around this code (lower camel case), thus `datasetLifetime`?